spring-cloud-task-core/pom.xml

```diff
@@ -22,6 +22,11 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-aop</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+		</dependency>
```

Comment:
We should look at what these starters bring in.  We probably want to be a bit more precise about what dependencies we're adding to the project.